app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,27 @@
+class InvitationsController < ApplicationController
+  def new
+    @invitation = Invitation.new
+  end
+
+  def create
+    user = User.find_by(inv_params)
+    event_id = cookies[:event_id]
+    cookies[:event_id] = nil
```

Comment:
- [ ] Why are you getting the event_id from the cookies when you have params? You can pass it as a parameter the same way you are passing the user name.